typing/btype.ml

```diff
@@ -470,7 +470,8 @@ let type_iterators_without_type_expr =
     | Named (_, mt) -> it.it_module_type it mt
   and it_module_type it = function
       Mty_ident p
-    | Mty_alias p -> it.it_path p
+    | Mty_static_alias p -> it.it_path p
+    | Mty_transparent p -> it.it_path p
```

Comment:
Can be slightly cleaned by keeping only one call to `it.it_path` like it was written before.